components/footer/footer.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import useTheme from '../use-theme'
+import FooterGroup from './footer-group'
+import FooterLink from './footer-link'
+import FooterColumn from './footer-column'
+import { FooterContext } from './footer-context'
+
+interface Props {
+  light?: boolean
+  subFooter?: string | React.ReactNode
+  ariaLabel?: string
+  maxWidth?: string
+  breakPoint?: string
```

Comment:
Maybe we also need to be compatible with the imperative API.
```
<Footer data={[]} />
```

Reply:
Would try to work on this!